src/Selenium2Library/keywords/_formelement.py

```diff
@@ -180,6 +180,26 @@ def choose_file(self, locator, file_path):
                         % file_path)
         self._element_find(locator, True, True).send_keys(file_path)
 
+    def choose_multiple_files(self, locator, basePath="",  *files):
+        """Inputs the `files` into file input field found by `locator`.
+        This Keyword extend the Choose Files option to allow user to select multiple files
+        (input type=files multiple).
+        The file paths need to be specified for each file.
+        Alternative is use basePath argument and then only files name.
+        The file specified with `file_path` must be available on the same host
+        where the Selenium Server is running.
+        Example:
+        | Choose Multiple Files | my_upload_field | /home/user/files/  |  trades.csv,trades.xsl,trades.doc |
```

Comment:
I am pretty sure that in the example `trades.csv,trades.xsl,trades.doc` is not correctly formatted. I think it should be 
```
| Choose Multiple Files | my_upload_field | /home/user/files/  |  trades.csv | trades.xsl | trades.doc |
```

Also fix the example in line 193.